src/prod_components/userEditingForm.js

```diff
@@ -69,7 +69,8 @@ class UserEditingForm extends React.Component {
   }
 
   handleColourChange = color => {
-    this.props.setUserColour( color.hex )
+    // Removing live colour update for the moment since it cause issues
+    // this.props.setUserColour( color.hex )
```

Comment:
I would appreciate more information on what issues it causes :)

I think you've done the right choice disabling it for now. Maybe create a GitHub issue to track the problem and implement a solution?